report/thesis.tex

```diff
@@ -105,10 +131,9 @@ \chapter{Tendermint protocol overview and the fork accountability problem}
 
 ## Agreement property
 
-We limit the discussion to the agreement property only because its proof can be useful to better understand the fork accountability problem. 
-We invite interested readers to read the official paper for a more formal, complete and rigorous proof of all the properties listed above.
+We present the agreement property of the Tendermint protocol because it is important to properly frame the fork accountability problem. In this section we also give a simple proof of the agreement property and show that it is not possible to violate the agreement property in Tendermint if there are at most *f* faulty processes.
 
-In this section we will give a simple proof of the agreement property for the Tendermint algorithm and show why the correct majority assumption given before (*n > 3f*, or simply *n = 3f + 1*) is important for ensuring this property.
+### Idea behind the proof
 
 The key idea behind the proof of the agreement property is that any two sets of *2f + 1* processes have at least one correct process in common.
```

Comment:
This should go below the sentence "we assume n = 3f + 1"

Reply:
True, fixed